.buildkite/longruns/pipeline.yml

```diff
@@ -64,3 +64,14 @@ steps:
           slurm_time: 24:00:00
         env:
           JOB_NAME: "longrun_ssp_bw_rhoe_equil_highres"
+      
+      - label: ":computer: aquaplanet equilmoist clearsky radiation + prognostic edmf diffusion only + 0M microphysics"
+        command:
+          - srun julia --project=examples examples/hybrid/driver.jl --config_file $CONFIG_PATH/$$JOB_NAME.yml --job_id $$JOB_NAME
+        artifact_paths: "$$JOB_NAME/output_active/*"
+        agents:
+          slurm_ntasks: 64
+          slurm_nodes: 4
```

Comment:
This will be much slower than choosing 56 tasks on 1 node

Reply:
I think I usually need to wait for very long (more than a few hours) if I ask for 1 node.